contracts/helpers.cairo

```diff
@@ -187,7 +192,7 @@ func _get_premia_before_fees{
         assert_nn(total_premia_before_fees);
     }
 
-    return (total_premia_before_fees,);
+    return (total_premia_before_fees=total_premia_before_fees);
```

Comment:
Doesn't matter, but, do you maybe know if there is actually a difference between these two approaches?